src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -101,7 +101,7 @@ public class GetAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         /// <summary>
         /// Friendly Name of the item, Applicable only in case of Azure File Share item.
         /// </summary>
-        [Parameter(Mandatory = false, Position = 7, HelpMessage = ParamHelpMsgs.Item.FriendlyName)]
+        [Parameter(Mandatory = false, HelpMessage = ParamHelpMsgs.Item.FriendlyName)]
```

Comment:
You will need to update the markdown help for this cmdlet to reflect these changes